Frontend/ui-driver/src/App.purs

```diff
@@ -121,7 +121,7 @@ defaultGlobalState = GlobalState{
 , noInternetScreen : {}
 , popUpScreen : PopUpScreenData.initData
 , driverRideRatingScreen : DriverRideRatingScreenData.initData
-, appUpdatePopUpScreen : {version : 1}
+, appUpdatePopUpScreen : {version : 1, appUpdateOrInaccourateTimeAndDate : ""}
```

Comment:
@ManavGuptaJuspay can you please correct the spelling of this variable or change the name